app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="auth_pages">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email address" %>
+    </div>
+
```

Comment:
- [ ] Dear Team, You have done an impressive job with the functionality, I'm only concerned your design for the login and sign-up does not match the wireframe at this point. Remember, the views in devise for the registration and login are expected to be modified by the student to match the wireframes and their styling. As you can see in the airframe the login is at the center of the page the buttons are aligned equally with the input fields and the title `Log in` is not displayed.

### Your design 
![Screen Shot 2023-08-30 at 8 29 34 PM](https://github.com/Microverse-Fullstack-Program/rails-blog-app-v2/assets/69077061/b1d3e4f5-96a8-47fd-abd2-31905fa73344)
### The wireframe 

![](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png?raw=true)